src/api-reference/expense/expense-report/v4.workflows.markdown

```diff
@@ -3,309 +3,181 @@ title: Workflows v4
 layout: reference
 ---
 
-{% include prerelease.html %}
+The Workflows v4 API can be used to Approve or Send Back an expense report that is in the workflow. This API can be used only for normal approval steps currently and not for Cost Object Approval steps(since there is no API that provides the currentProcessInstanceId which is required to approve cost objects) 
 
-Description of API, including business purpose and most common use case.
+> **Limitations**: This API is only available to partners who have been granted access by SAP Concur. Access to this documentation does not provide access to the API.
 
-> **Limitations**: This section is optional. Which limitations the user may encounter, can include feature gaps from previous versions, limitations on access tokens, or support limitations for certain scenarios. If API only works in certain data centers, mention that here.
-
-* [Process Flow](#process-flow)
+* [Prior Versions](#Prior-versions)
 * [Products and Editions](#products-editions)
 * [Scope Usage](#scope-usage)
 * [Dependencies](#dependencies)
 * [Access Token Usage](#access-token-usage)
+* [Approve or Send Back a Report](#approve-or-send-back-a-report)
 * [Create a resource](#create-resource)
```

Comment:
Can you please remove Create a resource and Obtain a representation from the toC?